modules/ROOT/pages/runtime-monitoring.adoc

```diff
@@ -232,13 +120,65 @@ Sample extract of exposed Prometheus data:
  # TYPE tomcat_servlet_request_max_seconds gauge
  tomcat_servlet_request_max_seconds{name="default",} 0.0
  tomcat_servlet_request_max_seconds{name="dispatcherServlet",} 0.104
- # HELP tomcat_threads_config_max_threads
- # TYPE tomcat_threads_config_max_threads gauge
- tomcat_threads_config_max_threads{name="http-nio-8080",} 200.0
- # HELP tomcat_sessions_expired_sessions_total
- # TYPE tomcat_sessions_expired_sessions_total counter
- tomcat_sessions_expired_sessions_total 0.0
- # HELP tomcat_sessions_active_max_sessions
- # TYPE tomcat_sessions_active_max_sessions gauge
- tomcat_sessions_active_max_sessions 0.0
```

Comment:
why removing those lines

Reply:
IMO It was a little too much for a sample